components/brave_rewards/browser/net/network_delegate_helper.cc

```diff
@@ -18,6 +22,7 @@
 #include "net/base/upload_bytes_element_reader.h"
 #include "net/base/upload_data_stream.h"
 #include "net/url_request/url_request.h"
```

Comment:
shouldn't we be removing this header if we're untying from net::URLRequest?

Reply:
there are still some deps in this file, will fix later.